monitoring/airgap/airgap-tempo-values.yaml

```diff
@@ -0,0 +1,4 @@
+image:
```

Comment:
Looking at the [Temp Helm chart, ](https://github.com/grafana/helm-charts/blob/6e2172f08b5539384734a42262f42836b8075fc5/charts/tempo/values.yaml#L16) it looks like the correct key  for this is NOT **image.repository** but **tempo.repository**.  Or, am I mis-reading things?